fileio/ft_filetype.m

```diff
@@ -1456,10 +1486,6 @@
   type = 'maus_textgrid';
   manufacturer = 'MAUS/WebMAUS';
   content = 'segmented text';
-elseif filetype_check_extension(filename, '.fcsv')
```

Comment:
It is not clear to me why this piece of code has been deleted

Reply:
Hi @robertoostenveld, No, we didn't intend to change other functions, except ft_read_header, ft_read_event and ft_read_data in this PR.  I guess this issue is because we sent the PR on Apr. 17th, and there are modification of the master branch of Fieldtrip since then.